Dockerfile

```
@@ -29,3 +29,4 @@ FROM gcr.io/distroless/static:latest
 WORKDIR /
 COPY --from=builder /workspace/secrets-manager .
 ENTRYPOINT ["/secrets-manager"]
+USER 1000
```

I would suggest using user `nobody` here instead, it's not clear what UID 1000 might be actually linked to on the host.

```suggestion
USER nobody
```